mail/opensmtpd/Portfile

```
@@ -0,0 +1,114 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        opensmtpd opensmtpd 6.8.0p1-rc1
+categories          mail
+platforms           darwin
+license             ISC
+maintainers         debian.org:rak
+description         secure, reliable, lean, and easy-to configure SMTP server
+long_description    The OpenSMTPD server seeks to be \
+                    \n* as secure as possible, and uses privilege separation to mitigate \
+                    \n  possible security bugs \
+                    \n* as reliable as possible: any accepted email must not be lost \
+                    \n* lean: it covers typical usage cases instead of every obscure one \
+                    \n* easy to configure, with a configuration syntax reminiscent of the OpenBSD \
+                    \n  Packet Filter's (PF) \
+                    \n* fast and efficient: it can handle large queues with reasonable performance
+homepage            https://www.opensmtpd.org
+
+github.tarball_from archive
+
+checksums           rmd160  186ee26a72f63aa3058bf9990a5a9db12f8ac2ae \
+                    sha256  e35a2954dad002ffaf2060358aa97e5268dbe653e22cdd5ebf7addb1dcabdcdb \
+                    size    447879
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:bison \
+                    port:libtool
+
+depends_lib         port:libevent \
+                    port:openssl
+
+startupitem.create  yes
+startupitem.name    OpenSMTPD
+startupitem.autostart   yes
+startupitem.executable  ${prefix}/sbin/smtpd
+
+add_users           _smtpd group=_smtpd realname=OpenSMTP\ Daemon \
+                    _smtpq group=_smtpq realname=OpenSMTP\ Queue
+
+set sysconfdir ${prefix}/etc/${name}
+
+post-extract {
```

Patching should happen in the patch phase.
```suggestion
post-patch {
```